### ISSUE TITLE
Unable to export from a specific figma file

### ISSUE THREAD

**Issue by smhmd** [marcomontalbano/figma-export#81]
The file in question (https://www.figma.com/file/zsNpuxeNsk1LvApsDUHRZu) is a duplicate of a [Community file](https://www.figma.com/community/file/866420896007969754)

Running 
```bash
npx -p @figma-export/cli -p @figma-export/output-components-as-svg figma-export components zsNpuxeNsk1LvApsDUHRZu -O @figma-export/output-components-as-svg
```

It either hangs or fails on `preparing components`.
![image](https://user-images.githubusercontent.com/46059092/111919359-de51fd80-8a89-11eb-80c5-9ae909d8ab67.png)

Thanks for the package!

**Comment by marcomontalbano** [comment]
Hi @smhmd,
thanks for opening this well-detailed issue.

It looks like the payload is too large (too many icons in a single request).
Probably I can try splitting the single request into different requests to reduce the payload.

I'll let you know as soon as a fix is ready.

**Comment by marcomontalbano** [comment]
Hi @smhmd,
I just released v3.2.1 which solves this issue.

You can try again by running 
```sh
npx -p @figma-export/cli -p @figma-export/output-components-as-svg figma-export components zsNpuxeNsk1LvApsDUHRZu -O @figma-export/output-components-as-svg
```

If something goes wrong please double-check if the downloaded version is the right one
```sh
npx -p @figma-export/cli figma-export version

# should print out something like "@figma-export/cli/3.2.1 ..."
```

Let me know!
